metadata/src/test/java/org/apache/kafka/controller/ClusterControlManagerTest.java

```diff
@@ -272,8 +273,16 @@ public void testRegistrationWithIncorrectClusterId() {
                 (short) 1));
     }
 
+    private static Stream<Arguments> metadataVersions() {
+        return Stream.of(
+                MetadataVersion.IBP_3_3_IV2,
+                MetadataVersion.IBP_3_3_IV3,
+                MetadataVersion.latest()
```

Comment:
Maybe we want to add in `MetadataVersion.LATEST_PRODUCTION` as well?

Reply:
Added